domains/krvishal.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/thegeekdex",
+    "owner": {
+        "username": "thegeekdex",
+        "email": "",
+        "discord": "Vishal S.#6993"
+    },
+    "record": {
+         "A": ["3.108.152.235"]
```

Comment:
Please complete your website.

Reply:
its complete, but can't be proxified without domain.

Reply:
@TheGeekDex 
![image](https://user-images.githubusercontent.com/87287585/196026163-b4639bcd-5c98-43c0-9fb9-10178f8c1a86.png)

Reply:
nothing is in construction, just need to update that in database. you can see, everything is working fine